src/Constants.ts

```diff
@@ -13,4 +13,10 @@ export const CheckpointPrefix = "#!%!";
 export const CheckpointSuffix = "!%!#";
 
 export const BlockTags = ["div", "p", "ul", "li", "h1", "h2", "h3"];
-export const Hardbreaks = ["br", "hr", "tr"];
\ No newline at end of file
+export const Hardbreaks = ["br", "hr", "tr"];
+
+export enum NodeType {
```

Comment:
Let's use the NF convention for Enums where they're always named in the plural form.